### DISCUSSION TITLE
Problem on GitHub student pack

### DISCUSSION THREAD

**Discussion by Ouarghii** [community/community#63850] [GitHub Education]
### Hi everyone,

But I can get the student pack github
I get this message :

Hey Raslen Ouarghi,Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Aug 13, 2023 for @Ouarghii.The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect email domain, please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.You appear not to be near any campus location for the school you have selected. Your school-provided academic affiliation document must indicate virtual learning if you are remote.Please enable the application to discover your location from the browser pop-up to provide an accurate location. If you are using a VPN, please disable it when you reapply. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect campus location, please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.
--

Hey Raslen Ouarghi,

Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Aug 13, 2023 for [@Ouarghii](https://github.com/Ouarghii).

The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect email domain, please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.
You appear not to be near any campus location for the school you have selected. Your school-provided academic affiliation document must indicate virtual learning if you are remote.Please enable the application to discover your location from the browser pop-up to provide an accurate location. If you are using a VPN, please disable it when you reapply. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect campus location, please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.

**Comment by GhostOf0days** [comment]
Hello. Please DO NOT post ANY private data to this discussion board. I don't remember if you are able to delete these posts, so please contact [Github Support](https://support.github.com/request?tags=docs-generic) and ask them to remove the post for you.

**Comment by safnimj** [comment]
You can try with your Following letter of your Institution
